os_migrate/galaxy.yml

```diff
@@ -19,7 +19,7 @@ tags:
 dependencies:
   community.crypto: '>=1.4.0'
   community.general: '<5.0.0'
-  openstack.cloud: '<2.0.0'
+  openstack.cloud: '==2.0.0'
```

Comment:
I think we may want go unpin and go with `>=2.0.0` and only pin to a specific version if we find that there is something that breaks us? I think the SDK should now be pretty stable and i'd hope that would somewhat translate to the Ansible collection too...